README.md

```diff
@@ -1,3 +1,24 @@
-# Superset
+# Readme
 
 [Apache Superset](https://superset.apache.org/) Docker image with additional libraries used with Medic's CHT deployments. This follows Superset's [advice about using a custom image](https://superset.apache.org/docs/installation/docker-builds/#building-your-own-production-docker-image).
+
+# Docker Image
+This docker image currently comes with following packages installed that are used in generally most of the superset deployments medic is supporting. 
+
+- `psycopg2-binary`: To connect to postgres databases
+- `Authlib`: To support OAuth login
+- `openpyxl`: To support uploading excel files for analysis
+- `Pillow`: For Alert and Report
+- `playwright`: For taking screenshots for Alerts & Reports
+
+## Usage
+To use this version of superset on your deployments, update your `values.yaml` to point to this superset image repo instead of `apachesuperset.docker.scarf.sh/apache/superset`
```

Comment:
```suggestion
To use this version of superset on your deployments, update your Helm `values.yaml` to point to this superset image repo instead of `apachesuperset.docker.scarf.sh/apache/superset`
```